### PR TITLE
chore: create notification customization pages

### PR DESCRIPTION
**Pull request by themonster2015**
## Description

* Create notification customization pages when first opening the notification app.
## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/notifications/src/components/newComponents/WelcomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  const goToNextStep = () => {
+    // navigate to step 2
+    return navigateTo?.({
+      appName: '@akashaorg/app-notifications',
+      getNavigationUrl: () => routes[CUSTOMIZATION_STEP_TWO],
+    });
+  };
+
+  const goToNotificationsPage = () => {
+    // @TODO: go to notification listing page.
```
Comment (SeverS) [inline ui/apps/notifications/src/components/newComponents/WelcomePage.tsx:39]: is this route unknown yet?

File: ui/apps/notifications/src/routes.ts [hunk 2]
```diff
@@ -1 +1,11 @@
 export const rootRoute = '/@akashaorg/app-notifications';
+
+export const CUSTOMIZATION_STEP_ONE = 'CustomizationStepOne';
+export const CUSTOMIZATION_STEP_TWO = 'CustomizationStepTwo';
+export const CUSTOMIZATION_STEP_THREE = 'CustomizationStepThree';
+
+export default {
+  [CUSTOMIZATION_STEP_ONE]: `/onboarding/step-one`,
```
Comment (SeverS) [inline ui/apps/notifications/src/routes.ts]: can these routes be more explicit? what is happening in step-one for example?

File: ui/apps/notifications/src/components/newComponents/WelcomePage.tsx (hunk 1 above)
Comment (themonster2015) [reply ui/apps/notifications/src/components/newComponents/WelcomePage.tsx:39]: Hi Sever. I'm working on the component for this route.

File: ui/apps/notifications/src/routes.ts (hunk 2 above)
Comment (themonster2015) [reply ui/apps/notifications/src/routes.ts]: I have changed the naming to make them more explicit.

**Review by SeverS — DISMISSED** [review]

**Review by SeverS — APPROVED** [review]

**Pull request merged**